src/ui/server/controllers/insurance/policy-and-export/single-contract-policy/save-and-back/index.ts

```diff
@@ -29,19 +27,13 @@ export const post = async (req: Request, res: Response) => {
     if (hasFormData(req.body)) {
       const validationErrors = generateValidationErrors(req.body);
 
-      const populatedData = mapSubmittedData(req.body);
-
-      let saveResponse;
+      const saved = await mapAndSave.policyAndExport(req.body, application, validationErrors);
 
-      if (validationErrors) {
-        saveResponse = await save.policyAndExport(application, populatedData, validationErrors.errorList);
-      } else {
-        saveResponse = await save.policyAndExport(application, populatedData);
+      if (saved) {
+        return res.redirect(`${INSURANCE_ROOT}/${referenceNumber}${ROUTES.INSURANCE.ALL_SECTIONS}`);
```

Comment:
can we add this route as a const since it's being used twice?

Reply:
And all sections could be dereferenced on line 7

Reply:
We can't declare it outside of the function as it needs the reference number param. Improved the dereferencing - think it's nice now